install/db/achecker_schema.sql

```diff
@@ -308,7 +308,7 @@ CREATE TABLE `themes` (
   `title` varchar(80) NOT NULL DEFAULT '',
   `version` varchar(10) NOT NULL DEFAULT '',
   `dir_name` varchar(20) NOT NULL DEFAULT '',
-  `last_updated` date DEFAULT NOT NULL,
+  `last_updated` date NOT NULL,
```

Comment:
Please merge in the master that has the latest changes of your previous pull request. This change, as well as the change in `install/db/achecker_upgrade_1.4_to_1.5.sql` seems from an older version of your previous pull request.

Reply:
The changes only affected the ``` themes``` table

Reply:
If it is ok with you, I would send a different pull request to fix the issue

Reply:
Sure. Probably continue to use ACHECKER-3 ticket to fix this issue. Thanks.

Reply:
Ok, would do that.